drv/cosmo-hf/src/hf.rs

```diff
@@ -35,6 +39,17 @@ impl ServerImpl {
             Ok(())
         }
     }
+
+    fn flash_base(&self) -> u32 {
+        match self.dev {
+            HfDevSelect::Flash0 => 0,
+            HfDevSelect::Flash1 => SLOT_SIZE_BYTES,
```

Comment:
it occurs to me that we _could_ be cute about it and have an enum in this crate whose variants are represented by the values 0 and `SLOT_SIZE_BYTES`,  like
```rust
enum CosmoDevSelect {
    Flash0 = 0,
    Flash1 = SLOT_SIZE_BYTES,
}
```
and then just use `self.dev as u32`, and convert between the API `HfDevSelect` at the API boundary, so that individual reads/writes don't have to  do this conversion. But, honestly? This is simpler and that's probably not worth the trouble...